packages/components/src/inputs/src/SearchField.module.css

```diff
@@ -0,0 +1,57 @@
+.hop-SearchField  {
+    --hop-SearchField-gap: var(--hop-space-stack-xs);
+    --hop-SearchField-default-inline-size: 20rem;
+    --hop-SearchField-color: var(--hop-neutral-text-weak);
+    --hop-SearchField-padding: var(--hop-space-inset-md);
+
+    display: flex;
+    flex-direction: column;
+    gap: var(--hop-SearchField-gap);
+    inline-size: var(--hop-SearchField-default-inline-size);
+}
+
+.hop-SearchField--fluid  {
+    inline-size: 100%;
+}
+
+.hop-SearchField__Label {
+    order: 0;
+
+    font-family: var(--hop-heading-xs-font-family);
+    font-size: var(--hop-heading-xs-font-size);
+    font-weight: var(--hop-heading-xs-font-weight);
+    line-height: var(--hop-heading-xs-line-height);
+    color: var(--hop-SearchField-color);
+}
+
+.hop-SearchField__prefix {
+    color: var(--hop-SearchField-color);
+}
+
+.hop-SearchField__InputGroup {
+    order:1;
+    inline-size: 100%;
+    padding-inline: var(--hop-SearchField-padding);
+}
+
+.hop-SearchField--clearable .hop-SearchField__InputGroup {
+    padding-inline-end: 0;
```

Comment:
you should split --hop-SearchField-padding into --hop-SearchField-padding-inline-end and --hop-SearchField-padding-inline-start and --hop-SearchField-padding-block.
that way you can set --padding-inline-end to --hop-SearchField-padding-inline-end inside .hop-SearchField--clearable.
Then the selector here would only need to be .hop-SearchField__InputGroup
which will have padding-inline-end: var(--padding-inline-end);